The DRBD module does not compile without that 

Those definitions are part of the original RHEL kernel. Not sure
why openvz patches removes that.

--- ../linux-2.6-2.6.32/include/linux/idr.h	2013-11-14 11:18:46.000000000 +0100
+++ linux-2.6.32-358.23.2.el6/include/linux/idr.h	2013-09-14 10:52:55.000000000 +0200
@@ -42,6 +42,12 @@
 #define MAX_ID_BIT (1U << MAX_ID_SHIFT)
 #define MAX_ID_MASK (MAX_ID_BIT - 1)
 
+/* Leave the possibility of an incomplete final layer */
+#define MAX_LEVEL (MAX_ID_SHIFT + IDR_BITS - 1) / IDR_BITS
+
+/* Number of id_layer structs to leave in free list */
+#define IDR_FREE_MAX MAX_LEVEL + MAX_LEVEL
+
 struct idr_layer {
 	unsigned long		 bitmap; /* A zero bit means "space here" */
 	struct idr_layer	*ary[1<<IDR_BITS];
